### PR TITLE
[Example App] Make some screens responsive on small screens

### PR DESCRIPTION
**Pull request by Hamza-Nasser**
I have made some screens responsive to small screens.

- [done ] I have updated `auto_suggest_box.dart, navigation_view.dart, progress_indicators.dart, tiles.dart, main.dart` with my changes to the sizes of widgets to make it responsive.
- [done ] I have run "dart format ." on the project - done
- [done ] I have added/updated relevant documentation

### CODE REVIEW

File: example/lib/screens/forms/auto_suggest_box.dart [hunk 1]
```diff
@@ -35,8 +41,10 @@ class _AutoSuggestBoxPageState extends State<AutoSuggestBoxPage>
         subtitle(content: const Text('A basic AutoSuggestBox')),
         CardHighlight(
           child: Row(children: [
+            // Either Expanded widget or just hard coded size like this
             SizedBox(
-              width: 350.0,
+              // width: 350.0,
+              width: smallScreenWidth ? 250 : 350.0,
```
Comment (bdlukaa) [inline example/lib/screens/forms/auto_suggest_box.dart:47]: I would say to use a `ConstrainedBox` with the `maxWidth` set to `350`

File: example/lib/screens/surface/tiles.dart [hunk 2]
```diff
@@ -82,40 +90,43 @@ ListView.builder(
         ),
         CardHighlight(
           child: Row(crossAxisAlignment: CrossAxisAlignment.start, children: [
-            Container(
-              height: 400,
-              width: 350,
```
Comment (bdlukaa) [inline example/lib/screens/surface/tiles.dart:87]: Again, I think a `ConstrainedBox` with `maxWidth` of `350.0` fits it better here

File: example/test/widget_test.dart
Comment (bdlukaa) [inline example/test/widget_test.dart:1]: Remove unused file

File: example/lib/screens/surface/tiles.dart (hunk 2 above)
Comment (Hamza-Nasser) [reply example/lib/screens/surface/tiles.dart:87]: I've tried this approach, but it didn't work as expected. No responsiveness.
If you have a better solution just give me a hint.

File: example/lib/screens/forms/auto_suggest_box.dart (hunk 1 above)
Comment (Hamza-Nasser) [reply example/lib/screens/forms/auto_suggest_box.dart:47]: I've tried this approach, but it didn't work as expected. No responsiveness.
If you have a better solution just give me a hint.

File: example/lib/screens/surface/tiles.dart (hunk 2 above)
Comment (Hamza-Nasser) [reply example/lib/screens/surface/tiles.dart:87]: Again, the same idea.

File: example/test/widget_test.dart
Comment (Hamza-Nasser) [reply example/test/widget_test.dart:1]: Ok.

**Pull request closed without merging**